onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -388,4 +394,34 @@ public void checkEditOnlineWithEditPermission() {
     addUsers.deleteUser(userC_name);
     manageLogInOut.signOut();
   }
+
+
+
+  @Test
+  /**
+   * Save_version_drawer_US01
+   * Check that "Add new version" button existing and that the drawer is opened
+   */
+  public void checkOpeningDrawerVersions() {
+    String document = "OO_test";
+    String extension = ".docx";
+    String userName = PLFData.DATA_NAME_ROOT;
+    info("Edit Online from document from AS");
+    activityStream.uploadFileFromAS(document + extension);
+    homePagePlatform.refreshUntil($(byXpath(ELEMENT_CHECK_FILE_IS_UPLOADED.replace("$doc", document + extension))), visible, timeout);
+    onlyOfficeActivityStream.editOnlineFromAS();
+    switchTo().window(1);
+    onlyOfficeEditingPage.checkOpeningDocumentWithEditOnline(document, extension, userName);
+    ELEMENT_OPEN_DRAWER_BUTTON.hover();
+    $(".tooltip.fada.bottom.in").should(visible);
+    ELEMENT_OPEN_DRAWER_BUTTON.waitUntil(visible, timeout).click();
```

Comment:
If the hover has been done 2 lines above, it means the button is already there. IMO we should put the waitUntil also in the line with the hover to make sure it is well visible. Maybe it is the cause of not seeing the tooltip ? (I did a manuel test on acceptance, I can see it, and I can open the drawer)